Docker/build/WikibaseBundle/LocalSettings.d.template/CirrusSearch.php

```diff
@@ -1,8 +1,6 @@
 <?php
 
 // https://www.mediawiki.org/wiki/Extension:CirrusSearch
-if ( getenv('MW_ELASTIC_HOST') !== false ) {
-    wfLoadExtension( 'CirrusSearch' );
-}
+wfLoadExtension( 'CirrusSearch' );
```

Comment:
is this revert wanted?

Reply:
As I see it ether we have wmde.5 and wmde.6 have the same internals (revert needed) or we make them different.
I think making them different would lead to more confusion, so I would hole back this cirrus search change until after this release is done.